### ISSUE TITLE
getMap() returns null after 6.1.1 upgrade

### ISSUE THREAD

**Issue by 7091lapS** [visgl/react-map-gl#1313]
I tried upgrading from 5.0.4 to 6.1.1 and now I can't seem to get a map object from the getMap function on the ref, it just returns null. Downgrading solves the issue. I'm confused because the ref is being set correctly, it's the getMap() function that fails to return the actual map object. This is my code:

```
  componentDidMount() {
    const map = this.mapRef.getMap();

    this.setState({
      map,
    ...
  }

render() {
    const { map } = this.state;
   ...
   return(
      <div className={styles.mapWrapper}>
        <ReactMapGL
          onLoad={() => this.onLoad()}
          ref={ref => this.mapRef = ref}

     ...
```

Any idea why that might be happening?

**Comment by daniela-green** [comment]
Exact same thing happens to me as well after upgrading. 
worked until now and after upgrading the **getMap()** returns null.

**Comment by Pessimistress** [comment]
Try 5.3.2/6.1.2

**Comment by daniela-green** [comment]
fixed with 5.3.2, thanks!

@Pessimistress
Similar to whats written above, im using:
**macOs 10/ Chrome 88**

**Comment by 7091lapS** [comment]
works on 6.1.2, thanks
